cmd/reviewdog/main.go

```diff
@@ -596,10 +597,19 @@ func gitlabClient(token string) (*gitlab.Client, error) {
 const defaultGitLabAPI = "https://gitlab.com/api/v4"
 
 func gitlabBaseURL() (*url.URL, error) {
-	baseURL := os.Getenv("GITLAB_API")
-	if baseURL == "" {
+	gitlabApi := os.Getenv("GITLAB_API")
+	gitlabV4Url := os.Getenv("CI_API_V4_URL")
+
+	var baseURL string
+	switch true {
```

Comment:
Can you use if-else clause? It's interesting, but simple if-else should be better.

Alternatively, you can create another function which returns URL string and use early return.

Reply:
I thought about the function first, but then it seemed weird to me to have the current gitlabBaseURL to just be a wrapper for `url.Parse`. I have switched to the if-else chain.